yarn.lock

```diff
@@ -2,70 +2,58 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.12.11":
-  version "7.12.11"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
-  integrity sha512-Zt1yodBx1UcyiePMSkWnU4hPqhwq7hGi2nFL1LeA3EUl+q2LQx16MISgJ0+z7dnmgvP9QtIleuETGOiOH1RcIw==
+"@bazel/typescript@^5.5.3":
+  version "5.5.3"
```

Comment:
Can you try going back to the previous version? 5.2.0?

Reply:
The errors in buildkite are along the lines of:
```
@bazel/typescript/index.bzl:23:6: cannot load '@rules_nodejs//nodejs/private:ts_config.bzl': no such file
```
so maybe something changed in 5.5.3 compared to 5.2.0?

Reply:
Ah, didn't notice that bazel specific dep. Downgraded now.

Reply:
That seems to help, at least it is another error now:

```
bazel-out/k8-fastbuild/bin/tests/typescript_keywords_generated.ts(6,39): error TS2307: Cannot find module './test_dir/typescript_include_generated.js' or its corresponding type declarations.
--
  | bazel-out/k8-fastbuild/bin/tests/typescript_keywords_generated.ts(7,26): error TS2307: Cannot find module './test_dir/typescript_transitive_include_generated.js' or its corresponding type declarations.
  | (13:37:53) INFO: Elapsed time: 39.561s, Critical Path: 17.75s
```